web/packages/core/texts/ru-RU/messages.ftl

```diff
@@ -14,6 +14,15 @@ enable-hardware-acceleration = Похоже, что аппаратное уск
 view-error-details = Сведения об ошибке
 open-in-new-tab = Открыть в новой вкладке
 click-to-unmute = Включить звук
+clipboard-message-title = Копирование и вставка в Ruffle
+clipboard-message-description =
+    { $variant ->
+       *[unsupported] Ваш браузер не поддерживает полный доступ к буферу обмена.
+        [access-denied]  Ваш браузер не разрешает доступ к буферу обмена.
```

Comment:
I don't think this translation is right. Usually it's the user who denies the access, not the browser. However, this string should be written in passive voice in order not to blame the user of doing something wrong. So instead of "You have denied access" or "The browser has denied access", it should be "Access has been denied".

Reply:
Well, it could be argued that the dogma of "the browser is the **user** agent" goes both ways. If I tell it to block ads, it should absolutely do that. But if it refused to grant clipboard access, it's because I didn't authorize it, not due to its own fault.

Reply:
Anyway, I recommend just getting this in (it's close enough), and moving this issue over to CrowdIn to be fixed later.

Reply:
That's alright, I wanted to mention this issue on Crowdin but I just don't understand their UI...

Reply:
To be honest, I don't entirely either... 😅 I think any issues I've left so far were more from the perspective of a translator, towards source string managers, and not the other way around, like I intended - but oh well, at least the info is there, which is the point.